src/badguy/zeekling.cpp

```diff
@@ -22,6 +22,7 @@
 #include "math/random.hpp"
 #include "object/player.hpp"
 #include "sprite/sprite.hpp"
+#include "audio/sound_manager.hpp"
```

Comment:
Please put the includes in alphabetical order, not counting the one referencing the hpp file.  Here, you just slapped it on at the end when it should really go not there.